README.Rmd

```diff
@@ -1,3 +1,7 @@
+---
```

Comment:
vcr uses knitr for building readme https://github.com/ropensci/vcr/blob/master/Makefile#L39 - i assume this yaml is an rmarkdown thing?

Reply:
Does the yaml break the make workflow? If not can I keep it for being able to use the unit button?